variants/rak4631_epaper/variant.h

```diff
@@ -126,7 +126,7 @@ static const uint8_t SCK = PIN_SPI_SCK;
  * eink display pins
  */
 
-#define PIN_EINK_EN (32 + 2) // (0 + 2) Note: this is really just backlight power
+#define PIN_EINK_EN (0 + 2) // (0 + 2) Note: this is really just backlight power
```

Comment:
I propose to remove that define alltogether. Should still work. 0+2 just points somehwhere random. see my findings in the main comments.